### PR TITLE
O3DE website Documentation

### PR DESCRIPTION
**Pull request by aFinchy**
Created 

## GitHub markdown

This manual is produced to assist members who are interested in establishing GitHub repositories or maintaining current ones and seeking to explore their true capabilities, along with step-by-step instructions and illustrative examples.

[GitHub Markdown](/markdown/github-markdown/README.md)

## Blog Process Guides

These resources are available to offer support and direction on optimal approaches for constructing and sustaining blogs for the O3DE Website. This document will be regularly revised to reflect any modifications made over time.

[Blog Process Guide](/markdown/o3de-website/README.md)<br>
[O3DE Blog Cheatsheet](/markdown/o3de-website/website-markdown/README.md)<br>
[Templates and Examples](/markdown/o3de-website/website-markdown/files)

## O3DE Website Maintenance

These resources offer support in generating code, banners, and color schemes for the website. Additionally, they provide instructions on updating images, adding new foundation members, and revising different sections of the o3de website.

[O3DE Website Updates](/markdown/o3de-website/website-updates/README.md)

### CODE REVIEW

File: markdown/o3de-website/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Blog Process
+
+ This guide provides guidance on the process of publishing a blog on the O3DE website. This document covers selecting a topic, identifying an author, designing the blog, and publishing on site. Additionally, it offers troubleshooting tips in case any issues arise during the blog's publication. 
+
+ To ensure a smooth publishing process, it is crucial to keep all relevant parties informed about the scheduled release date and any updates related to it. This includes the author of the blog and other stakeholders, such as the Marketing Committee. Clear communication with all parties involved can prevent any confusion or delays in the publishing process.
+
+Checklist
+
```
Comment (chanmosq) [inline markdown/o3de-website/README.md:8]: We can start using some headers here to organize the documents. I also suggest including a introductory sentence for the section.

```suggestion
## Checklist

The overall process for publishing a blog is broken down to the following items:

```

File: markdown/o3de-website/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        - [ ] 7.0. Check to make sure blog is live on day its supposed to. 
+        - [ ] 7.1. Contact Docs if blog has not gone live
+    - [ ] Social Media posting
+        
+        
+
+Terms:
+PR - Pull Request
+SIG - Special Interest Group
+TSC - Technical Steering Committee
```
Comment (chanmosq) [inline markdown/o3de-website/README.md:32]: - Again, we can organize this into it's own section. It allows readers to easily jump to this section when needed. 
- Also provide an short introductory sentence to the section
- Nit: Style formatting for the terminology
- Links for additional information would also be good here
```suggestion
## Terminology

This document uses the following terminology:

- **PR**:  [Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
- **SIG**: [Special Interest Group](https://github.com/o3de/community#special-interest-groups)
- **TSC**: [Technical Steering Committee](https://github.com/o3de/tsc)
```

File: markdown/o3de-website/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
```
Comment (chanmosq) [inline markdown/o3de-website/README.md:34]: Another section header
```suggestion
## Details

This section provides additional details for each item in the checklist.

```

File: markdown/o3de-website/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    - [ ] 3.2. Plan soft date for publish
+- [ ] 4. PR Stage
+    - [ ] 4.1. Markdown design
+    - [ ] 4.2. Publish date finalized
+    - [ ] 4.3. Social Media
+    - [ ] 4.4. PR Stage Timeline
+    - [ ] 4.5. Testing the blog on local machine
+- [ ] 5. Social Media
+- [ ] 6. Blog goes live
+    - [ ] Follow-up
```
Comment (chanmosq) [inline markdown/o3de-website/README.md:22]: Should this be **7**? The points below it are numbered as 7.0 and 7.1.

**Review by chanmosq — CHANGES_REQUESTED** [review]
Left a few high-level suggestions for o3de-website/README.md to start. Will continue review of this

File: markdown/o3de-website/README.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        
+         To merge the blog into the main website, at least two individuals must approve the PR. 
+        
+        It is recommended to allow for 1-2 weeks (7 to 14 days) for the blog to be merged from the time of submission into PR. Rushing the process may lead to errors or missing the intended publication date.
+    5. **Testing how the blog displays**
+        If you want to test to see how the blog looks before it goes live. This is normally the best practice to ensure that everything renders properly, then you will need to either configure your machine to build the site locally or ask someone to build the site and send images over to you. 
+        
+        If testing locally, the best guide would be the up-to-date date here. https://github.com/o3de/o3de.org, Once you have a working build, to get the blog to show on the list, it would be required to run the following command: 
+        `Hugo Server -F`
+        This ensures that the build of the website is built in the future and will be able to view the blog as if it was publication day.
```
Comment (chanmosq) [inline markdown/o3de-website/README.md:76]: - A few rewording suggestions to eliminate fragmented sentences and improve flow and clarity
- Converted link to hyperlink for clarity. 
- Added more instruction on where the user can expect to see the blogs

```suggestion
        It's best practice to test how the blog looks before it goes live to ensure that everything renders properly. To do this you must either configure your machine to build the site locally or ask someone to build the site and send images over to you. 
        
        If testing locally, refer to instructions in the `o3de.org` repo, [Build the website locally](https://github.com/o3de/o3de.org). Once you have a working build, you can find the list of blogs at the `/blog` address of your local web server. For example, `http://localhost:1313/blog/`. However, to get the blog to show in the list, you must rerun with the following command: 
        `hugo server -f`
        This ensures that the build of the website shows posts that are scheduled for future publication, allowing you to view the blog as if it was publication day. 
```

**Review by chanmosq — CHANGES_REQUESTED** [review]
Great job on the Blog Process doc! I've left a some suggestions.

**I also have some comments about the overall info architecture:**
I'm not sure if there's a reason for the structure of the documentation in the `markdown/` folder, however I found it odd that they are all in separate folder's READMEs. The user would have to click into each folder to get to the document and may also have trouble finding the doc they need.  As an alternative solution, you might consider flattening all the files (no subfolders) in the `markdown/` folder, and renaming them accordingly.  For example, I think `o3de-website/README.md` should be renamed to `blog-processing.md`. That would help us navigate the folder better and find the file we're looking for.

**Comment by aFinchy** [comment]
Closed going to make a PR per Document

**Pull request closed without merging**